lib/helper.c

```diff
@@ -326,8 +329,18 @@ int fuse_main_real(int argc, char *argv[], const struct fuse_operations *op,
 
 	if (opts.singlethread)
 		res = fuse_loop(fuse);
-	else
-		res = fuse_loop_mt(fuse, opts.clone_fd);
+	else {
+		struct fuse_loop_config loop_config;
+		loop_config.clone_fd = opts.clone_fd;
+		if (opts.max_idle_threads == 0) {
```

Comment:
Ah, sorry. In that case simply move the default setting into `fuse_parse_cmdline`, after the `memset`.